### PR TITLE
set T2 with the correct value in ConfigurableFakeBackend

### LINKED ISSUE #8112: T2 of `ConfigurableFakeBackend` is not properly set
### Environment

- **Qiskit Terra version**: 0.20.2
- **Python version**: 3.7.13
- **Operating system**: Linux

### What is happening?

When `ConfigurableFakeBackend` is initialized with `qubit_t2`, an actual T2 value of the device is set based on `qubit_t1`.

### How can we reproduce the issue?

```python
from qiskit.test.mock.utils.configurable_backend import ConfigurableFakeBackend

backend = ConfigurableFakeBackend('my_fake', 1, qubit_t1=182, qubit_t2=135, std=0.0)  # Pass `std=0` for deterministic result.
print(backend.properties().t1(0), backend.properties().t2(0))
# output: 0.000182 0.000182
```

### What should happen?

The T2 value of the backend initialized like above should be generated based on `qubit_t2`.

### Any suggestions?

In [`qiskit/test/mock/utils/configurable_backend.py`](https://github.com/Qiskit/qiskit-terra/blob/e5087294007550ac3c2a3691e510de204fc98fac/qiskit/test/mock/utils/configurable_backend.py#L84), pass `qubit_t2` instead of `qubit_t1`.
Current implementation:
```diff
- qubit_t2 = np.random.normal(loc=qubit_t1 or 150.2, scale=std, size=n_qubits).tolist()
+ qubit_t2 = np.random.normal(loc=qubit_t2 or 150.2, scale=std, size=n_qubits).tolist()
```

And [unit test for `ConfigurableFakeBackend` with T2 set](https://github.com/Qiskit/qiskit-terra/blob/5ed5db90b7f4b5988647aa72384b7833ea1184a7/test/python/mock/test_configurable_backend.py#L38) does exist, but T2 is **NOT** asserted.
We should add assertions to check all attributes set through `__init__()`.

### PR DESCRIPTION
**Pull request by pradkrish**
### Summary
Fixes #8112 by setting T2 in `ConfigurableFakeBackend` with the input value

### Details and comments
Unit tests added to verify it works

### CODE REVIEW

File: test/python/mock/test_configurable_backend.py [hunk 1]
```diff
@@ -56,6 +56,8 @@ def test_set_parameters(self):
                 self.assertEqual(properties.backend_name, "Tashkent")
                 self.assertEqual(len(properties.qubits), n_qubits)
                 self.assertEqual(len(properties.gates), n_qubits)
+                self.assertAlmostEqual(properties.t1(0), 9.9e-5, places=7)
+                self.assertAlmostEqual(properties.t2(0), 14.6e-5, places=7)
```
Comment (jakelishman) [inline test/python/mock/test_configurable_backend.py:60]: A very minor nitpick, but when we're doing scientific notation with SI prefixes, we'd usually use powers of 1,000 to do the scaling - `99e-6` seconds is clearly (once you're used to the convention) 99µs, but I have to think for a little bit what `9.9e-5` means.

**Review by jakelishman — COMMENTED** [review]
This looks sensible, thanks.  Please could you add a "bugfix" release note with a short sentence saying that setting the T_2 time in `ConfigurableFakeBackend` is now fixed?

File: test/python/mock/test_configurable_backend.py (hunk 1 above)
Comment (pradkrish) [reply test/python/mock/test_configurable_backend.py:60]: Ack

**Review by jakelishman — APPROVED** [review]
Looks good to me, thanks for the change!

**Pull request merged**